app/data/container-build/cerc-fixturenet-eth-geth/Dockerfile

```diff
@@ -1,8 +1,12 @@
 FROM skylenet/ethereum-genesis-generator@sha256:210353ce7c898686bc5092f16c61220a76d357f51eff9c451e9ad1b9ad03d4d3 AS ethgen
 
+FROM golang:1.18-alpine as delve
```

Comment:
Not important for this builder base image, but if possible we would like to standardize on Ubuntu base images, not alpine.

Reply:
I was deliberately matching it to the version used to build geth: https://github.com/cerc-io/go-ethereum/blob/v1.10.26-statediff-v4/Dockerfile#L7

I was afraid of running into glibc / musl issues if they didn't match.

Reply:
Let me add a comment....